### PR TITLE
chore(STONEO11Y-232): Dashborad editor access for development env

### PR DESCRIPTION
**Pull request by ShakedAviv50313**
* Grant Grafana dashborad editor access for development environment

### CODE REVIEW

File: components/monitoring/grafana/development/kustomization.yaml [hunk 1]
```diff
@@ -2,3 +2,10 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../base
+patches:
+  - path: auto-assign-role-patch.yaml
+    target:
+      name: grafana-oauth
+      kind: Grafana
+
+
```
Comment (eisraeli) [inline components/monitoring/grafana/development/kustomization.yaml:11]: Redundant line at the end of the file (you have 2 newlines)

Comment (ShakedAviv50313) [reply components/monitoring/grafana/development/kustomization.yaml]: Fixed that, thanks!

**Comment by eisraeli** [comment]
LGTM.

**Review by eisraeli — APPROVED** [review]
LGTM

**Comment by ShakedAviv50313** [comment]
@gbenhaim can you please have a look?

**Comment by avi-biton** [comment]
Hi @adambkaplan @johnmcollier , can you please take a look and approve? :)

**Comment by ShakedAviv50313** [comment]
Hi @dirgim, can you please have a look and approve?

**Pull request merged**